.github/workflows/review.yaml

```diff
@@ -0,0 +1,105 @@
+#
+# Check for the Squid Project Merge Process criteria
+# published at https://wiki.squid-cache.org/MergeProcedure#votes
+#
+name: PR Approval
+
+on:
+  schedule:
+    - cron: "10 1 * * *" # once a day
+
+  pull_request:
+    # test PRs targeting this branch code
+    branches: [ "master" ]
+
+  # allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  # Cancel ongoing tests in case of push to the same PR or staging branch,
+  # but let previous master commit tests complete.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+  # required for use of gh CLI tool
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+
+  Review:
+    if: ${{ github.repository }} == 'squid-cache/squid'
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: read
+      pull-requests: write
+
+    steps:
+      - name: Checkout Squid sources
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Reviewer Veto
+        # Any Reviewer needing changes has priority hold.
+        id: core-veto
+        run: |
+          gh pr view ${{ github.event.pull_request.number }} --json latestReviews \
+            --template '{{range .latestReviews}}{{.state}}{{end}}' |
+            grep CHANGES_REQUESTED && false
+
+      - name: Fast Track Approval
+        # Unanimous Approval from Squid Core team overrides 48hr hold check.
```

Comment:
3 from 4 is not unanimous, please correct

Reply:
done